community-plugins.json

```diff
@@ -5788,5 +5788,12 @@
         "author": "Ryan Gomba",
         "repo": "ryangomba/obsidian-todo-sort",
         "branch": "main"
+    },
+    {
+        "id": "obsidian-file-chucker",
```

Comment:
Similarly, you don't need to include the word "obsidian" in your ID.

Reply:
it's just "file-chucker" now.